src/Contracts/Contracts.ts

```diff
@@ -17,4 +17,10 @@ export interface IKubeService {
     getDaemonSets(): Promise<K8sTypes.V1DaemonSetList>;
 
     getStatefulSets(): Promise<K8sTypes.V1StatefulSetList>;
+}
+
+export enum KubeImage {

```

Comment:
make this 

export enum KubeImage {
    zeroData = "zeroData",
    zeroResults = "zeroResults",
    zeroWorkloads = "zeroWorkloads"
}

so in future we will not have problems with moving the items up and down

Reply:
Got it. Great feedback!